ush/parsing_namelists_CICE.sh

```diff
@@ -37,374 +38,72 @@ if (( $(( NY_GLB % NPY )) == 0 )); then
-   f_snoice        = 'mdhxx'
-   f_dsnow         = 'mdhxx'
-   f_melts         = 'mdhxx'
-   f_meltt         = 'mdhxx'
-   f_meltb         = 'mdhxx'
-   f_meltl         = 'mdhxx'
-   f_fresh         = 'mdhxx'
-   f_fresh_ai      = 'x'
-   f_fsalt         = 'mdhxx'
-   f_fsalt_ai      = 'x'
-   f_fbot          = 'mdhxx'
-   f_fhocn         = 'mdhxx'
-   f_fhocn_ai      = 'x'
-   f_fswthru       = 'x'
-   f_fswthru_ai    = 'x'
-   f_fsurf_ai      = 'x'
-   f_fcondtop_ai   = 'x'
-   f_fmeltt_ai     = 'x'
-   f_strairx       = 'mdhxx'
-   f_strairy       = 'mdhxx'
-   f_strtltx       = 'x'
-   f_strtlty       = 'x'
-   f_strcorx       = 'x'
-   f_strcory       = 'x'
-   f_strocnx       = 'mdhxx'
-   f_strocny       = 'mdhxx'
-   f_strintx       = 'x'
-   f_strinty       = 'x'
-   f_taubx         = 'x'
-   f_tauby         = 'x'
-   f_strength      = 'x'
-   f_divu          = 'mdhxx'
-   f_shear         = 'mdhxx'
-   f_sig1          = 'x'
-   f_sig2          = 'x'
-   f_sigP          = 'x'
-   f_dvidtt        = 'mdhxx'
-   f_dvidtd        = 'mdhxx'
-   f_daidtt        = 'mdhxx'
-   f_daidtd        = 'mdhxx'
-   f_dagedtt       = 'x'
-   f_dagedtd       = 'x'
-   f_mlt_onset     = 'mdhxx'
-   f_frz_onset     = 'mdhxx'
-   f_hisnap        = 'x'
-   f_aisnap        = 'x'
-   f_trsig         = 'x'
-   f_icepresent    = 'x'
-   f_iage          = 'x'
-   f_FY            = 'x'
-   f_aicen         = 'x'
-   f_vicen         = 'x'
-   f_vsnon         = 'x'
-   f_snowfracn     = 'x'
-   f_keffn_top     = 'x'
-   f_Tinz          = 'x'
-   f_Sinz          = 'x'
-   f_Tsnz          = 'x'
-   f_fsurfn_ai     = 'x'
-   f_fcondtopn_ai  = 'x'
-   f_fmelttn_ai    = 'x'
-   f_flatn_ai      = 'x'
-   f_fsensn_ai     = 'x'
-/
-
-&icefields_mechred_nml
-   f_alvl         = 'x'
-   f_vlvl         = 'x'
-   f_ardg         = 'x'
-   f_vrdg         = 'x'
-   f_dardg1dt     = 'x'
-   f_dardg2dt     = 'x'
-   f_dvirdgdt     = 'x'
-   f_opening      = 'x'
-   f_ardgn        = 'x'
-   f_vrdgn        = 'x'
-   f_dardg1ndt    = 'x'
-   f_dardg2ndt    = 'x'
-   f_dvirdgndt    = 'x'
-   f_krdgn        = 'x'
-   f_aparticn     = 'x'
-   f_aredistn     = 'x'
-   f_vredistn     = 'x'
-   f_araftn       = 'x'
-   f_vraftn       = 'x'
-/
-
-&icefields_pond_nml
-   f_apondn       = 'x'
-   f_apeffn       = 'x'
-   f_hpondn       = 'x'
-   f_apond        = 'mdhxx'
-   f_hpond        = 'mdhxx'
-   f_ipond        = 'mdhxx'
-   f_apeff        = 'mdhxx'
-   f_apond_ai     = 'x'
-   f_hpond_ai     = 'x'
-   f_ipond_ai     = 'x'
-   f_apeff_ai     = 'x'
-/
-
-&icefields_drag_nml
-   f_drag         = 'x'
-   f_Cdn_atm      = 'x'
-   f_Cdn_ocn      = 'x'
-/
-
-&icefields_bgc_nml
-/
-eof
+# Prepare local variables for use in ice_in_template from UFSWM
+# setup_nml section
+local SYEAR=${current_cycle:0:4}
+local SMONTH=${current_cycle:4:2}
+local SDAY=${current_cycle:6:2}
+local SECS=${sec}
+local DT_ICE=${ICETIM}
+local NPT=${npt}  # TBT [To be templated in ice_in_template; UFSWM sets to 999]
```

Comment:
@DeniseWorthen 
Can we work to make this (and other `TBT`) variables templated in the ufs-weather-model RTs?

Reply:
This https://github.com/ufs-community/ufs-weather-model/pull/1998 should help.